ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
Agreed. Should we cross that bridge when we have tests that actually use the test database?